client/src/pages/signUpPage.tsx

```diff
@@ -66,6 +76,16 @@ const SignUpPage: React.FC = () => {
             placeholder="create password"
             required
           />
+          <FormLabel>Confirm Password</FormLabel>
+          <TextField
+            id="password"
+            type="password"
+            size="small"
+            variant="outlined"
+            placeholder="enter your password"
```

Comment:
The placeholder should be either 're-enter you password' or 'confirm password' .